<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Reset default Qt styles */
* {
    margin: 0;
    padding: 0;
    border: 0;
    font: inherit;
}

/* Set a global font for your application */
body {
    font-family: Helvetica, Arial, sans-serif;
}

/* Style for main application window */
#mainWindow {
    background-color: #f4f4f4;
    padding: 10px;
}

/* Style for buttons */
QPushButton {
    background-color: #3498db;
    color: #fff;
    padding: 10px 20px;
    border: none;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #2980b9;
}

/* Style for text input fields */
QLineEdit {
    padding: 8px;
    border: 1px solid #ccc;
    border-radius: 5px;
}

/* Style for labels */
QLabel {
    font-weight: bold;
    color: #333;
}

/* Style for list widgets */
QListWidget {
    border: 1px solid #ccc;
    border-radius: 5px;
    background-color: white;
}

QListWidget::item {
    padding: 10px;
}

QListWidget::item:hover {
    background-color: #f0f0f0;
}

/* Style for progress bars */
QProgressBar {
    text-align: center;
}

/* Style for checkboxes */
QCheckBox {
    padding: 5px;
}

/* Style for radio buttons */
QRadioButton {
    padding: 5px;
}

/* Style for tabs */
QTabWidget::pane {
    border: 1px solid #ccc;
    border-top: none;
    border-radius: 5px;
    background-color: white;
}

QTabBar::tab {
    padding: 10px 20px;
    border: 1px solid #ccc;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTabBar::tab:selected {
    background-color: #3498db;
    color: #fff;
    border: 1px solid #2980b9;
}

/* Style for menus and menu items */
QMenu {
    background-color: white;
    border: 1px solid #ccc;
}

QMenu::item {
    padding: 8px 16px;
}

QMenu::item:selected {
    background-color: #3498db;
    color: #fff;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0,0,0,1,20,1,20,1,20">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <widget class="QPushButton" name="import_btn">
          <property name="text">
           <string>Import</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_mixer_btn">
          <property name="text">
           <string>Mixer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Frequency</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="main_frequency_slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_normalized"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Fmax</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>8</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Noise</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="main_noise_slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1,1,1,5">
        <item>
         <widget class="QPushButton" name="fmax_1_btn">
          <property name="text">
           <string>Fmax</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fmax_2_btn">
          <property name="text">
           <string>2Fmax</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fmax_3_btn">
          <property name="text">
           <string>3Fmax</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fmax_4_btn">
          <property name="text">
           <string>4Fmax</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14" stretch=""/>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Original</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="main_widget_1" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 120, 215);
</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Reconstruction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="main_widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 120, 215);</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Error</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="main_widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 120, 215);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
